src/core/types/workspace.ts

```diff
@@ -19,4 +25,5 @@ export interface WorkspaceAttribute {
 
 export interface WorkspaceAttributeWithPermission extends WorkspaceAttribute {
   permissions?: Permissions;
+  permissionMode?: PermissionModeId;
```

Comment:
Q: Is this the permission of current user to the workspace?

Reply:
Yes, it is permission for current user which shows here 
![image](https://github.com/user-attachments/assets/09c3d72a-0383-4dd3-ab26-d84a40342d13)